internal/commands/commands.go

```diff
@@ -74,7 +74,20 @@ Realm app will be exported as well.`,
 				Help:        "",
 				Command:     &app.CommandInit{},
 			},
-			{
+			cli.CommandDefinition{
+				Use:         "create",
+				Display:     "app create",
+				Description: "Create a remote Realm app from your current working directory",
+				Help: `Creates a new Realm app in the cloud, and saves your configuration files on 
+a new disk. This command will create a new directory for your project. 
+To initialize a Realm app in your current working directory, use ‘app init’.
+
+You can specify a ‘From’ input to create a Realm app from either a template, 
```

Comment:
i know this is from the doc, but we should tweak this a little to avoid mentioning `templates` (as thats not supported/implemented yet).  I also think using `'--from'` instead of `'From'` makes more sense in this context, thoughts?

Reply:
agree